packages/meteor-gazelle-createdby/lib/tests/server.js

```diff
@@ -0,0 +1,36 @@
+Tinytest.add('Createdby - On create, records creator (also as updater)',
+                                                        function (test) {
+  // Arrange
```

Comment:
@daveyshindig Yeah they do but we're not in a chaotic world. The code is readable and understandable without the extra comments.